src/pnpq/devices/odl_ozoptics_650ml.py

```diff
@@ -4,63 +4,135 @@
 import serial
 from serial import Serial
 from pnpq.devices.optical_delay_line import OpticalDelayLine
-import time
+import time, logging
+from pnpq.errors import (
+    DeviceDisconnectedError,
+    OdlMoveNotCompleted,
+    OdlHomeNotCompleted,
+    OdlGetPosNotCompleted,
+    OdlMoveOutofRangeError,
+)
+
 
 
 class OdlOzOptics(OpticalDelayLine):
+    max_move: int
+    """maximum length for moving odl from 0-position, the total path length will be 2x(max_move)"""
```

Comment:
As far as I know, docstrings should not be used to document individual variables.

Please see PEP 257 for details on how to use docstrings. https://peps.python.org/pep-0257/

These should be ordinary comments. They should be on the line above the variable and start with `#`. They should not use `"""`.